config/settings.yml

```diff
@@ -845,6 +845,12 @@ theaters:
       - name: Buckhorn
         lat: 34.7773774204513
         lon: -117.896163742621
+      - name: Bullseye NTTR Blue
```

Comment:
kannst du das NTT bitte noch wegnehmen? der Text ist sonst zu lang in der MDC und steht dann über oder wird zu klein